[2021-01-28 13:42:20,033] {taskinstance.py:826} INFO - Dependencies all met for <TaskInstance: ge_bash_op_dag.drop_monthly_data 2021-01-28T12:41:26.759507+00:00 [queued]>
[2021-01-28 13:42:20,040] {taskinstance.py:826} INFO - Dependencies all met for <TaskInstance: ge_bash_op_dag.drop_monthly_data 2021-01-28T12:41:26.759507+00:00 [queued]>
[2021-01-28 13:42:20,040] {taskinstance.py:1017} INFO - 
--------------------------------------------------------------------------------
[2021-01-28 13:42:20,040] {taskinstance.py:1018} INFO - Starting attempt 1 of 1
[2021-01-28 13:42:20,040] {taskinstance.py:1019} INFO - 
--------------------------------------------------------------------------------
[2021-01-28 13:42:20,046] {taskinstance.py:1038} INFO - Executing <Task(SnowflakeOperator): drop_monthly_data> on 2021-01-28T12:41:26.759507+00:00
[2021-01-28 13:42:20,049] {standard_task_runner.py:51} INFO - Started process 3349 to run task
[2021-01-28 13:42:20,057] {standard_task_runner.py:75} INFO - Running: ['airflow', 'tasks', 'run', 'ge_bash_op_dag', 'drop_monthly_data', '2021-01-28T12:41:26.759507+00:00', '--job-id', '144', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/ge_bash_op_dag.py', '--cfg-path', '/var/folders/s2/xy3ssc8j3ks3whp2f_b8708c0000gp/T/tmpkeoso723']
[2021-01-28 13:42:20,059] {standard_task_runner.py:76} INFO - Job 144: Subtask drop_monthly_data
[2021-01-28 13:42:20,097] {logging_mixin.py:103} INFO - Running <TaskInstance: ge_bash_op_dag.drop_monthly_data 2021-01-28T12:41:26.759507+00:00 [running]> on host 1.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.ip6.arpa
[2021-01-28 13:42:20,129] {taskinstance.py:1230} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=sara@acloudfrontier.com
AIRFLOW_CTX_DAG_OWNER=sara
AIRFLOW_CTX_DAG_ID=ge_bash_op_dag
AIRFLOW_CTX_TASK_ID=drop_monthly_data
AIRFLOW_CTX_EXECUTION_DATE=2021-01-28T12:41:26.759507+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-01-28T12:41:26.759507+00:00
[2021-01-28 13:42:20,129] {snowflake.py:119} INFO - Executing: ['delete from MONTHLY_DATA;']
[2021-01-28 13:42:20,136] {base.py:65} INFO - Using connection to: id: SNOWFLAKE_CONN_DEMO. Host: dp13486.eu-west-1.snowflakecomputing.com, Port: None, Schema: PUBLIC, Login: SARA, Password: XXXXXXXX, extra: XXXXXXXX
[2021-01-28 13:42:20,138] {connection.py:215} INFO - Snowflake Connector for Python Version: 2.3.8, Python Version: 3.8.6, Platform: macOS-10.16-x86_64-i386-64bit
[2021-01-28 13:42:20,139] {connection.py:768} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2021-01-28 13:42:20,140] {connection.py:784} INFO - Setting use_openssl_only mode to False
[2021-01-28 13:42:21,375] {cursor.py:537} INFO - query: [ALTER SESSION SET autocommit=True]
[2021-01-28 13:42:21,570] {cursor.py:560} INFO - query execution done
[2021-01-28 13:42:21,572] {dbapi.py:180} INFO - Running statement: delete from MONTHLY_DATA;, parameters: None
[2021-01-28 13:42:21,573] {cursor.py:537} INFO - query: [delete from MONTHLY_DATA;]
[2021-01-28 13:42:22,160] {cursor.py:560} INFO - query execution done
[2021-01-28 13:42:22,162] {dbapi.py:186} INFO - Rows affected: 10000
[2021-01-28 13:42:22,163] {connection.py:438} INFO - closed
[2021-01-28 13:42:22,256] {connection.py:441} INFO - No async queries seem to be running, deleting session
[2021-01-28 13:42:22,421] {taskinstance.py:1135} INFO - Marking task as SUCCESS. dag_id=ge_bash_op_dag, task_id=drop_monthly_data, execution_date=20210128T124126, start_date=20210128T124220, end_date=20210128T124222
[2021-01-28 13:42:22,446] {taskinstance.py:1195} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-01-28 13:42:22,490] {local_task_job.py:118} INFO - Task exited with return code 0
